### ISSUE TITLE
Seasons progression on server

### ISSUE THREAD

**Issue by Beckazon** [lucaargolo/fabric-seasons#172]
Hi there,
is there a way to only let seasons progress on a server when a player is online? So it stays the same season when nobody is online?
Else you have a farm in summer. Logout for a day and next time it's winter and you can forget the farm.
Hope there's a configs to edit or so.
Thanks in advance

**Comment by GeekyBakerBen** [comment]
> Hi there, is there a way to only let seasons progress on a server when a player is online? So it stays the same season when nobody is online? Else you have a farm in summer. Logout for a day and next time it's winter and you can forget the farm. Hope there's a configs to edit or so. Thanks in advance

Hey, I don't know if you're still looking for this, but I found another mod that does this called "Ready Player Fun". It'll pause server progression when nobody is on. Just testing it out myself now.

**Comment by Beckazon** [comment]
Hey. Thanks for the tip. Is that a server-side only mod then?

**Comment by GeekyBakerBen** [comment]
> Hey. Thanks for the tip. Is that a server-side only mod then?

Yes it is. From the brief testing I did, it seemed to work out pretty well for that use case. However, what I've now decided to do with mine instead is edit the Seasons config directly and increased the seasonal length to be 3 real-life months instead, so it will stay Winter over the whole of actual winter, and let the server tick on as standard.

Obviously in both cases, this is only possible for me because I am the server op, so I can make these changes. If you're on another person's server, you'll need to get them to do this for you.

(Oh, and if you DO decide to change the config to increase the season length in the same way, to save you some maths, 3 real-life months (90 days) is approximately 155520000 ticks in the game :D)
